### PR TITLE
Add test for `trivy fs`

### PR DESCRIPTION
**Pull request by nayuta**
Add test for `trivy fs` to check https://github.com/reviewdog/action-trivy/issues/17, the test was added.

Changes:
- multiple test was supported with GitHub Actions' matrix
  - You can put the test under `./testdata` directory with `./testdata/[trivy command]/[platform label]/[with_detections|without_detections]`
  - e.g. current terraform test is in `./testdata/config/terraform/with_detections` and `./testdata/config/terraform/without_detections`

### CODE REVIEW

**Comment by haya14busa** [comment]
Sorry for the late response. Can you resolve the conflicts?

**Comment by nayuta** [comment]
Thanks for your reply. However, the test on Windows failed accidentally. So, I'll check it this weekend.

**Comment by haya14busa** [comment]
I guess we can ignore the windows error for now. https://github.com/reviewdog/action-trivy/pull/51#issuecomment-2197947144

Is this review ready?

**Comment by nayuta** [comment]
> I guess we can ignore the windows error for now. https://github.com/reviewdog/action-trivy/pull/51#issuecomment-2197947144
> 
> 
> 
> Is this review ready? 

Yes, it's ready.

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -11,48 +11,35 @@ defaults:
...
-            exit 1
-          fi
+          # Find all the directories in the testdata directory
+          DIRS="$(find ./testdata -type d)"
+          for dir in ${DIRS}; do
+            if [ "$(echo "${DIRS}" | grep -c "${dir}" || true)" -le 1 ]; then
+              DIRS_RESULT="${DIRS_RESULT}${dir},"
+            fi
+          done
+          MATRIX_JSON=$(echo "${DIRS_RESULT}" | sed 's/,$//' | sed 's/\\n//g' | jq -R -s -c 'split(",") | [.[] | {dir: ., command: split("/")[2], type: split("/")[4], label: [split("/")[2],split("/")[3]] | join("/")}]')
```
Comment (haya14busa) [inline .github/workflows/tests.yml:31]: Can you add some comments? 
I don't quite understand what this script is doing.

**Comment by nayuta** [comment]
@haya14busa how do you think about this?

**Review by haya14busa — APPROVED** [review]
Thanks!

**Pull request merged**